samples/python_interop/generating_n_random_bits/RunGenerateRandom.py

```diff
@@ -0,0 +1,15 @@
+import qsharp
+
+qsharp.init(project_root=".")
+
+max = input("Enter the max number: ")
```

Comment:
To match the other suggestion
```suggestion
max = input("Enter the number of random bits to be generated: ")
```